plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1538,56 +1520,170 @@ public static function get_subscription_list_data() {
 				$subscription['version'] = $updates[ $subscription['product_id'] ]['version'];
 			}
 		}
+
+		// Sort subscriptions by name and expiration date.
+		usort(
+			$subscriptions,
+			function( $a, $b ) {
+				$compare_value = strcasecmp( $a['product_name'], $b['product_name'] );
+				if ( 0 === $compare_value ) {
+					return strcasecmp( $a['expires'], $b['expires'] );
+				}
+				return $compare_value;
+			}
+		);
+
+		// Add subscription install flags after the active and local data is set.
+		foreach ( $subscriptions as &$subscription ) {
+			$subscription['subscription_available'] = self::is_subscription_available( $subscription, $subscriptions );
+			$subscription['subscription_installed'] = self::is_subscription_installed( $subscription, $subscriptions );
+		}
+
 		// Break the by-ref.
 		unset( $subscription );
 
 		return $subscriptions;
 	}
 
+	/**
+	 * Check if subscription is available.
+	 *
+	 * @param array $subscription The subscription data.
+	 * @param array $subscriptions The list of subscriptions.
+	 * @return bool True if multiple licenses exist, false otherwise.
+	 */
+	public static function is_subscription_available( $subscription, $subscriptions ) {
+		if ( true === $subscription['active'] ) {
+			return false;
+		}
+
+		if ( true === $subscription['expired'] ) {
+			return false;
+		}
+
+		$product_subscriptions = wp_list_filter(
+			$subscriptions,
+			array(
+				'product_id' => $subscription['product_id'],
+				'active'     => true,
+			)
+		);
+
+		// If there are no active subscriptions for this product, then it's available.
```

Comment:
I've already forgotten what we're doing with the filter above. Is this the right description? Will subs that are active on other sites have `'active' => true`?

```suggestion
		// If there are no subscriptions for this product already active on this site, then it's available.
```